DEVELOPING.md

```diff
@@ -94,4 +94,8 @@ Add localized strings in `src/main/resources/messages/LibertyBundles_{locale}.pr
 All translatable text should be included in the message file(s). Message parameters should only contain language neutral text.
 
 ### Test Videos
-To record videos for all tests, not just the failed ones, you can create a `video.properties` file in the `src/test/resources` directory and add `video.save.mode=ALL` to that file.
\ No newline at end of file
+To record videos for all tests, not just the failed ones, you can create a `video.properties` file in the `src/test/resources` directory and add `video.save.mode=ALL` to that file.
+### Handling Out of Memory Errors in Build or Tests
+If you encounter an "OutOfMemoryError" during compilation or running tests, it may be due to insufficient heap space for the Java process. To resolve this:
+1. Open the `build.gradle` file.
+2. Increase the `memoryMaximumSize` in the `JavaCompile` task. For example: tasks.withType(JavaCompile) { options.forkOptions.memoryMaximumSize = "4g" // Increase to 4GB or more if needed }
```

Comment:
@staicy123 , Can you highlight this part `tasks.withType(JavaCompile) { options.forkOptions.memoryMaximumSize = "4g"` similar to what you did for `memoryMaximumSize` and `JavaCompile`